config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  # get 'root/index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
-end
+  root 'root#index'
+
```

Comment:
[OPTIONAL] Although your API endpoints work perfectly. It's always a great idea to make sure that you have no errors.

You have a routing error on your index page.

![image](https://user-images.githubusercontent.com/38088818/201467895-fd1f00a6-a341-4910-803f-937ca3698074.png)

That's because you do not have a `messages_controller.rb` outside of the API directory that should have the `index` method to route to when routing to 'root#index'. 
Also, the `index` action needs the `views/messages/index.html.erb` file to display something on the root (homepage).